src/articles/retour-paris-web-flo.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Retour sur Paris Web 2016
+author: Florent Berthelot
+date: 2016-10-31
+---
+
+Chaque année, et ce depuis 11 ans, a lieu à Paris une conférence majeure : **Paris Web**. La bien nommée tire sa particularité dans le rassemblement des développeurs et des UX/UI designers, bref de tous ceux qui font le web d'aujourd'hui. L'occasion idéale de s'éloigner un peu de ma mission actuelle de développeur back-end et de prendre du recul sur le net en général.
+
+De plus l'accent est mis sur l'accessibilité : tous les talks sont traduits en langue des signes, une vélotypie est proposée sur la grande scène et même le traiteur - très fameux par ailleurs, le meilleur congrès en terme de nourriture - respecte l'accessibilité et propose des emplois accessibles à tous. Je me demande d'ailleurs pourquoi les autres gros rendez-vous ne proposent pas de traduction en langue des signes. Réveillez-vous !
+
+Dans cet article, je vais faire un retour presque entièrement chronologique de mon premier Paris Web. N'hésitez pas à picorer les sujets qui vous intéressent. Si vous voulez approfondir un sujet, les liens vers les slides sont présents quand ceux-ci sont disponibles. *Je mettrai à jour cet article quand les vidéos seront disponibles.*
+
+## Jeudi
+
+### Anatomie d'une désintoxication au Web sous surveillance - Thibault Jouannic
+
+Nous le savons tous, Facebook, Google, Amazon, Apple (GAFA) et consorts sont partout sur le web, par le biais notamment de boutons *like*, de scripts d'analyse de trafic, ... Partout et sur quasiment toutes les pages du web ! Nous le savons tous, surtout nous qui travaillons dans le web et pourtant on ne fait rien et nous les intégrons même à nos page web.
+
+[Thibault Jouannic](https://twitter.com/thibaultj), dans sa [conférence de sensibilisation](https://www.miximum.fr/blog/conf-pw-2017/), nous invite à nous poser les bonnes questions. Pour lui – et je partage son avis – toutes les informations des utilisateurs sont sensibles, même celles qui ne le paraissent pas au premier abord. Il a pris son propre exemple : étant freelance, son adresse – qui est aussi celle de sa société – est affichée publiquement sur le web. Oui, mais sa femme travaille dans une prison et les détenus pourraient potentiellement retrouver leur adresse par ce biais. Cet exemple montre bien que des informations aussi "anodine" qu’une adresse peuvent être sensibles. Si toutes les informations sont sensibles, alors pourquoi les rendre accessibles à nous-même ou pire, aux GAFA ?
```

Comment:
IMO I find that an address seems to be a very sensitive piece of information, even without this context!